connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -337,6 +330,71 @@ object KustoWriter {
     CsvRowResult(fields.toArray, size + fields.size)
   }
 
+  private def convertOneField(row: InternalRow, schemaFields: Array[StructField], fieldIndexInRow: Int, timeZone: String) = {
```

Comment:
`convertField`

Reply:
reorder params. 
`row: InternalRow, fieldIndexInRow: Int, schemaFields: Array[StructField], , timeZone: String`

Reply:
also, isn't it kinda redundant to pass the entire schema fields array? you know what field you are operating on...

Reply:
I might not know scala enough, but shouldn't this also have a return type?